doc/qa_docs/scr/0.6.rst

```diff
@@ -0,0 +1,122 @@
+SCR Listing for ARMI version 0.6.0
+==================================
+
+This is a listing of all the Software Change Request (SCR) changes in the ARMI repository, as part
+of release number 0.6.0.
+
+Please note that the Software Test Report (STR) documents for all of the changes listed below will
+be updated as part of this release. This is a necessary part of every ARMI release, that the STRs
+are all updated.
+
+Below, this SCR is organized into the individual changes that comprise the net SCR for this release.
+
+
+Code Changes, Features
+^^^^^^^^^^^^^^^^^^^^^^
+
+The following software changes added new features, or made a meaningful to change to an existing one.
+
+.. list-table:: Code Changes, Features
+   :widths: 20 25 25 15 15 10 10
+   :header-rows: 1
+
+   * - Title
+     - Change
+     - | Impact on
+       | Requirements
+     - Author(s)
+     - Approver(s)
+     - Issue(s)
+     - PR(s)
+   * - TBD
+     - TBD
+     - TBD
+     - TBD
+     - TBD
+     - TBD
+     - TBD
+
+
+Code Changes, Bugs and Fixes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following software changes related to bugs or other fixes.
+
+.. list-table:: Code Changes, Bugs and Fixes
+   :widths: 20 25 25 15 15 10 10
+   :header-rows: 1
+
+   * - Title
+     - Change
+     - | Impact on
+       | Requirements
+     - Author(s)
+     - Approver(s)
+     - Issue(s)
+     - PR(s)
+   * - TBD
+     - TBD
+     - TBD
+     - TBD
+     - TBD
+     - TBD
+     - TBD
+
+
+Code Changes, Maintenance, or Trivial
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This selection collects more trivial software changes. Such a change might fall into a usual "maintenance" bin. It is probably part of the continuous and on-going technical debt efforts. But the change might be as minor as linting.
+
+.. list-table:: Code Changes, Maintenance, or Trivial
+   :widths: 20 25 20 15 15 10 10
+   :header-rows: 1
+
+   * - Title
+     - Change
+     - | Impact on
+       | Requirements
+     - Author(s)
+     - Approver(s)
+     - Issue(s)
+     - PR(s)
+   * - TBD
+     - TBD
+     - TBD
+     - TBD
+     - TBD
+     - TBD
+     - TBD
+
+
+Documentation-Only Changes
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The following changes below are changes to the SRSD, SDID, STR, or other SQA documentation.
+
+.. list-table:: Documentation-Only Changes
+   :widths: 20 25 25 15 15 10 10
+   :header-rows: 1
+
+   * - Title
+     - Change
+     - | Impact on
+       | Requirements
+     - Author(s)
+     - Approver(s)
+     - Issue(s)
+     - PR(s)
+   * - Adding an SCR section to the docs
+     - Adding a new process to ARMI. Instead of just a random "release notes" section, every ARMI PR will come with an SCR. This is part of the on-going quality work in ARMI.
+     - NA
+     - John Stilley
+     - Arrielle Opotowsky
+     - NA
+     - `PR#2090 <https://github.com/terrapower/armi/pull/2090>`_
+   * - TBD
+     - TBD
+     - TBD
+     - TBD
+     - TBD
+     - TBD
+     - TBD
```

Comment:
```suggestion
   * - TBD
     - TBD
     - TBD
     - TBD
     - TBD
     - TBD
     - TBD
   * - Adding an SCR section to the docs
     - Adding a new process to ARMI. Instead of just a random "release notes" section, every ARMI PR will come with an SCR. This is part of the on-going quality work in ARMI.
     - NA
     - John Stilley
     - Arrielle Opotowsky
     - NA
     - `PR#2090 <https://github.com/terrapower/armi/pull/2090>`_
   * - TBD
     - TBD
     - TBD
     - TBD
     - TBD
     - TBD
     - TBD
   * - TBD
     - TBD
     - TBD
     - TBD
     - TBD
     - TBD
     - TBD
   * - TBD
     - TBD
     - TBD
     - TBD
     - TBD
     - TBD
     - TBD
```